docs/conf.py

```diff
@@ -31,16 +31,16 @@
 
 # General information about the project.
 project = u'bok-choy'
-copyright = u'2015, EdX'
+copyright = u'2016, EdX'
```

Comment:
I don't really know enough about copyright ... are we actually supposed to change this each year?